src/components/Drawer/Drawer.scss

```diff
@@ -0,0 +1,87 @@
+.Drawer {
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    bottom: 0px;
+    right: 100px;
+    width: var(--drawer-width);
+    max-height: 100vh;
+    box-shadow:  0 5px 10px rgba(0,0,0,.1);
+    z-index: 100;
+    transform: translateY(100%) translateY(calc(-1 * var(--drawer-head-height)));
+    transition: var(--transition-duration);
+    transition-timing-function: cubic-bezier(.4,0,.2,1);
```

Comment:
I think it's better to replace it with `(0.4, 0, 0.2, 1)` for better readability